db/migrate/20170125015659_create_districts.rb

```diff
@@ -2,7 +2,7 @@ class CreateDistricts < ActiveRecord::Migration
 
   def change
     enable_extension "postgis"
-    enable_extension "pgrouting"
+    # enable_extension "pgrouting"
```

Comment:
I also thought your force push removed the commenting out of `pg_routing` so should check this as well.

Reply:
weird, I had thought so too. Maybe that was a different branch and I didn't notice? In any case, I fixed that in the newest commit.